includes/lib/transifex-live-integration-hreflang.php

```diff
@@ -164,6 +165,17 @@ public function render_hreflang() {
       $hreflang_out .= <<<XDEFAULT
 <link rel="alternate" href="$source_url" hreflang="x-default"/>\n
 XDEFAULT;
+    if (!$disable_canonical_urls) {
+      $canonical_url = home_url('/');
+      if (!empty($url_path)) {
+          $canonical_url = home_url($url_path);
+      }
+      $canonical_url = rtrim($canonical_url, '/') . '/';
+
+      $hreflang_out .= <<<CANONICAL
+  <link rel="canonical" href="$canonical_url"/>\n
+  CANONICAL;
+    }
```

Comment:
As discussed, let's better re-use the way that HREFLANG is constructed as in lines 158-164 above

Reply:
Updated as discussed!